### PR TITLE
Move trampolines

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/gtk-rs/gir/pull/792

### CODE REVIEW

File: src/auto/application.rs [hunk 1]
```diff
@@ -368,6 +398,12 @@ impl<O: IsA<Application>> GtkApplicationExt for O {
 
     #[cfg(any(feature = "v3_24", feature = "dox"))]
     fn connect_property_screensaver_active_notify<F: Fn(&Self) + 'static>(&self, f: F) -> SignalHandlerId {
+#[cfg(any(feature = "v3_24", feature = "dox"))]
+        unsafe extern "C" fn notify_screensaver_active_trampoline<P, F: Fn(&P) + 'static>(this: *mut gtk_sys::GtkApplication, _param_spec: glib_sys::gpointer, f: glib_sys::gpointer)
+        where P: IsA<Application> {
```
Comment (EPashkin) [inline src/auto/application.rs]: Wrong ident

Comment (GuillaumeGomez) [reply src/auto/application.rs]: Actually we don't need the condition anymore. I'll remove it its generation. :)

**Comment by GuillaumeGomez** [comment]
Ok, this one is ready.

**Comment by EPashkin** [comment]
👍

**Comment by sdroege** [comment]
:+1: and also for all the others. It's just code moved around after all :)

**Pull request merged**